SYCL/Plugin/interop-level-zero.cpp

```diff
@@ -53,6 +53,10 @@ int main() {
 
   backend_input_t<backend::ext_oneapi_level_zero, event> EventInteropInput = {
       ZeEvent};
+  // ZeEvent isn't owning the resource, we cannot transfer ownership that we
```

Comment:
So, the real L0 event resources was created by SYCL RT in https://github.com/intel/llvm-test-suite/pull/1032/files#diff-7d9caf352e8bfa25a4a5e0c96638dce519cf6058ecd8c6d5c1d218de88bd9e6bR20, and will be destroyed at `Event` dtor, right? Please add this into comment, if yes.